### ISSUE TITLE
Can't pip install after upgrading to El Capitan

### ISSUE THREAD

**Issue by jsma** [GrahamDumpleton/mod_wsgi#97]
I upgraded from Yosemite to El Capitan. I use pyenv for installing various versions of Python on my system. It took some yak shaving to get the Python installs working again (they couldn't find the openssl libs to build some packages) but now I hit an error when trying to ``pip install mod_wsgi``:

    ...
    In file included from src/server/mod_wsgi.c:21:
    In file included from src/server/wsgi_apache.h:38:
    In file included from /usr/include/apache2/httpd.h:44:
    In file included from /usr/include/apache2/ap_config.h:25:
    /usr/include/apache2/ap_hooks.h:39:10: fatal error: 'apr.h' file not found
    #include "apr.h"
             ^
    1 error generated.
    error: command 'clang' failed with exit status 1

As the error message indicates, ``apr.h`` does not exist in ``/usr/include/apache2``. Here's where that file may exist:

    $ sudo find / -name apr.h
    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/apr-1/apr.h
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift-migrator/sdk/MacOSX.sdk/usr/include/apr-1/apr.h
    /usr/include/apr-1/apr.h

**Comment by GrahamDumpleton** [comment]
What do you get when you run:

```
which apxs
```

and:

```
$ apxs -q INCLUDEDIR
/usr/include/apache2

$ apxs -q CPPFLAGS

$ apxs -q CFLAGS

$ apxs -q EXTRA_INCLUDES
-I/usr/include/apache2 -I. -I/usr/include/apr-1 -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.10.Internal.sdk/usr/local/include/

$ apxs -q EXTRA_CPPFLAGS
-DDARWIN -DSIGPROCMASK_SETS_THREAD_MASK -DDARWIN_10

$ apxs -q EXTRA_CFLAGS

```

These are values for 10.10, but need to see what is generated for 10.11.

**Comment by jsma** [comment]
Here's the output (as in your example, blank lines indicate no output):

    $ which apxs
    /usr/sbin/apxs
    $ apxs -q INCLUDEDIR
    /usr/include/apache2
    $ apxs -q CPPFLAGS

    $ apxs -q CFLAGS

    $ apxs -q EXTRA_INCLUDES
    -I/usr/include/apache2 -I. -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.Internal.sdk/usr/include/apr-1
    $ apxs -q EXTRA_CPPFLAGS
    -DDARWIN -DSIGPROCMASK_SETS_THREAD_MASK -DDARWIN_10 -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.Internal.sdk/usr/include/apr-1
    $ apxs -q EXTRA_CFLAGS

**Comment by GrahamDumpleton** [comment]
What is in the directory:

```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.Internal.sdk/usr/include/apr-1
```

In the past Xcode wasn't required, but now looks a bit like it might be.

Also, does ``/usr/include/apr-1`` exist?

**Comment by jsma** [comment]
Hmm, ``MacOSX10.11.Internal.sdk`` doesn't exist at all but this does (no 'Internal'):

    /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/

There is an ``apr-1`` directory under this location and yes, ``/usr/include/apr-1`` also exists.

**Comment by GrahamDumpleton** [comment]
Okay. Looks like I will have to upgrade my spare laptop to work out a fix. Apple breaks apxs in one way or another in every release. I haven't had to use workarounds though in past for how used in pip install, only for a configure/make install. This time it seems they have broken both. :-(

**Comment by jsma** [comment]
Sorry to be the bearer of bad news! Thanks for the prompt replies. This is not an emergency on my part, I only use this as a Django ``runserver`` replacement locally but can just use that in the meantime.

**Comment by GrahamDumpleton** [comment]
To get it going for now, use:

```
CFLAGS=-I/usr/include/apr-1 pip install mod_wsgi
```

**Comment by jsma** [comment]
That works, thanks!

**Comment by GrahamDumpleton** [comment]
Fixed in 4.4.14.
